File: bookapp_input_output/src/app/model/book.ts

export class Book {

    constructor(private _bookid: number,private _bookname: string,private _bookprice: number)
    {
   
    }

    public get bookprice(): number {
        return this._bookprice;
    }
    public set bookprice(value: number) {
        this._bookprice = value;
    }
    public get bookname(): string {
        return this._bookname;
    }
    public set bookname(value: string) {
        this._bookname = value;
    }
    public get bookid(): number {
        return this._bookid;
    }
    public set bookid(value: number) {
        this._bookid = value;
    }


}

File: ang_java_fullstack_example/restservice/src/main/java/com/angular/restservice/Book.java
package com.angular.restservice;

public class Book {
	
	
    private int id;
	private String bookname;
	private int bookprice;
	
	public Book(int bookId, String bookName, int bookPrice) {
		super();
		this.id = bookId;
		this.bookname = bookName;
		this.bookprice = bookPrice;
	}

	
	
	
	
	public int getId() {
		return id;
	}





	public void setId(int id) {
		this.id = id;
	}





	public String getBookname() {
		return bookname;
	}





	public void setBookname(String bookname) {
		this.bookname = bookname;
	}





	public int getBookprice() {
		return bookprice;
	}





	public void setBookprice(int bookprice) {
		this.bookprice = bookprice;
	}





	@Override
	public boolean equals(Object obj) {
		if (this == obj) {
			return true;
		}
		if (!(obj instanceof Book)) {
			return false;
		}
		Book other = (Book) obj;
		if (id != other.id) {
			return false;
		}
		
		return true;
	}

	@Override
	public String toString() {
		return "Book [bookId=" + id+ ", bookName=" + bookname + ", bookPrice=" + bookprice + "]";
	}
	
	

}
